### DISCUSSION TITLE
Database `mydb.public` does not exist on the database server at `localhost:5040`

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#7864] [Q&A]
I keep getting the error in the title above when I try to change my database, specifically "upsert".

**Comment by ryands17** [comment]
@Helixable 👋 

Could you create an issue [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with all the reproduction steps so that we can look into this.
